core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -341,7 +342,7 @@ object NextTurnAutomation {
     private fun chooseBeliefs(civInfo: CivilizationInfo, beliefContainer: BeliefContainer): HashSet<Belief> {
         val chosenBeliefs = hashSetOf<Belief>()
         // The 'continues' should never be reached, but just in case I'd rather have AI have a
-        // belief less than make the game crash. The 'continue's should only be reached whenever
+        // belief less than make the game crash. The 'continues' should only be reached whenever
```

Comment:
I actually disagree. One "continue", many "continue"s.

Reply:
I just adapted the combo to the one on the line above, so Studio would shut up. That was a waait - do I need to think about this? Nah, don't wanna - look there's a precedent so we don't have to think -> all in ~600ms. With 3000ms,  I'd tend to an approach like  "let's pretend markup works here even if it doesn't" and use ` to demark the keyword and 's do make plurals...    So many words for a measly comment...   Mebbeth I should turn off spellchecking inspection globally instead.